### DISCUSSION TITLE
is there a possibility to generate a plantuml file or an image?

### DISCUSSION THREAD

**Discussion by fatofit** [TNG/ArchUnit#1145] [Q&A]
Dear community and archunit developers/maintainers

I am looking for a way to work (the other way around) with PlantUML. 
For example, we would like to get the result of a test in a diagram or image.
I am talking about tests that describe the structure of the project. As an example, maybe slicing into packages or defining the layers and the allowed and unallowed accesses.

Very short question: Is this possible?

**Comment by hankem** [comment]
It isn't immediately obvious to me what you want to achieve. (This is why I forgot to reply to your question, sorry!)
Do you want to generate a package diagram highlighting violations of a layered architecture with access rules? 
I don't think that the results of an arbitrary `ArchTest` can be visualized in a canonical way.
